docs/app/insights.md

```diff
@@ -1,81 +1,199 @@
-Drag the panel to a new position and/or grab the corners/sides to resize it as needed.
+### Panel Header Options
 
-:::
+All 4 panel types let you set custom text to provide quick and clear context about what a panel represents. Panel Header
+options are exactly the same for all four panel types. They are also totally optional and can be hidden.
 
-## Duplicating a Panel
+![image of Header Options](../assets/insights-images/insights_panels_panel-header-options.gif)
 
-1. Click on the Insights module to go to the Dashboard listing
-2. Click the desired dashboard to view it
-3. Click the "Edit Panels" action button ("pencil" icon) in the header
-4. Hover over the desired panel, click the Context Menu button ("..." icon), and click "Duplicate"
-5. Reposition and resize as needed
-6. Click the "Save" action button in the page header
+- **Visible** - Determines if the header is visible or not
+- **Name** - Sets a name in the header area
+- **Icon** - Sets a small icon by the name
+- **Color** - Sets a color for the icon
+- **Note** - Sets a short description of the icon
 
-## Deleting a Panel
+### Label Overview
 
-1. Click on the Insights module to go to the Dashboard listing
-2. Click the desired dashboard to view it
-3. Click the "Edit Panels" action button ("pencil" icon) in the header
-4. Hover over the desired panel, click the Context Menu button ("..." icon), and click "Delete Panel"
-5. Click the "Save" action button in the page header
+As the name implies, this panel simply allows you to _label_ a group of other panels. Labels do not provide analytics.
+Labels simply help visually group the other panels (_which do provide analytics_) and give context to your dashboard.
+
+#### Label Panel Options
+
+![Static image of a label options](../assets/insights-images/insights_panels_label-options.gif)
+
+- **Label** - Sets the label text
+- **Color** - Sets the label text color
+
+### List Overview
+
+List panels display some number of items from a collection, _as a list!_ These items displayed can be filtered by other
+criteria as well. So for example, you could show the site's top 5 selling products, 3 worst performing sales regions, 10
+students with the highest GPA _that are also_ in intramural sports, or create any other kind of ranked list that your
+dataset supports.
+
+#### List Panel Options
+
+![Image of List panel options](../assets/insights-images/insights_panels_list-panel-options.gif)
+
+- **Collection** - Selects the collection to rank
+- **Limit** - Limits the number of items that will appear in the dashboard
+- **Sort Field** - Sets the collection field you wish to order results by.
+- **Sort Direction** - Sets items to display in ascending or descending order
+- **Display Template** - Sets which fields will appear on-screen to represent list items
+- **Filter** - Sets filters to only consider and rank items that meet some criteria
+
+### Metrics Overview
+
+Metrics panels allow you to aggregate numeric fields into a single number.
```

Comment:
Might need to update this, @jaycammarano just updated this to handle more than numbers. 🙂

Reply:
Also, we're speaking with devs, but a brief overview of what aggregate functions are and can do might be nice. Or at least a `(eg: SUM, AVERAGE, etc)`.

Reply:
Aaaand, `FIRST` and `LAST` aren't aggregate functions, so we should call those out. :)

Reply:
Yeah, I originally had one and it started to seem overwhelmingly large and like it belonged in more of a tutorial.
I will add something back in.

Reply:
Docs should be comprehensive in terms of what we offer (I'd even like to LIST all the functions we support here with what they do). We'll only switch to Tutorials when the topic becomes subjective or includes external software integrations.

Reply:
Wow, okay. I can do it! But listing it will be a heavy duty, long doc.... OR since there is repetition among panel types, and new panels types are coming in steadily, I may need to fundamentally restructure my approach.

Reply:
There are only 7-10 functions/aggregations... a short and simple bullet list with a short sentence on each is fine.

Reply:
> It smells to me like a new Aggregates doc may in order?

Makes sense to me. It's also noted under the API reference: https://docs.directus.io/reference/query/#aggregation-grouping, but I don't want to point less technical users there for this info 👍🏻

Reply:
> Aaaand, `FIRST` and `LAST` aren't aggregate functions, so we should call those out. :)

Ben, here is how I addressed that:
```
::: tip First, Last, Minimum and Maximum

First, Last, Minimum and Maximum are not aggregates in the purely mathematical sense of the word.
However, they are placed in this panel option because they return one single Item.

:::
```